tsl/src/compression/create.c

```diff
@@ -844,11 +854,10 @@ validate_existing_constraints(Hypertable *ht, CompressColInfo *colinfo)
 			attnums = (int16 *) ARR_DATA_PTR(arr);
 			for (j = 0; j < numkeys; j++)
 			{
-				FormData_hypertable_compression *col_def =
+				Form_hypertable_compression col_def =
```

Comment:
converting it from a pointer into a struct?
Won't that cause issues with the `Ensure` below?

Reply:
I am just using the existing definition rather than explicitly defining a pointer.
```c
typedef FormData_hypertable_compression *Form_hypertable_compression;
```